diff --git a/src/dolphinui.rc b/src/dolphinui.rc
--- a/src/dolphinui.rc
+++ b/src/dolphinui.rc
@@ -103,13 +103,12 @@
     <ToolBar noMerge="1" name="mainToolBar" >
         <text context="@title:menu">Main Toolbar</text>
         <Action name="go_back" />
         <Action name="go_forward" />
-        <Action name="view_settings" />
         <Action name="url_navigators" />
+        <Action name="toggle_search" />
         <Action name="split_view" />
-        <Action name="split_stash" />
-        <Action name="toggle_search" />
+        <Action name="view_settings" />
         <Action name="hamburger_menu" />
     </ToolBar>
     <ActionProperties scheme="Default">
